README.md

```diff
@@ -1,2 +1,69 @@
 # data-collection
+
 Data and automated tools for data collection
+
+# Requirements
+
+- The device must have [`ffmpeg`](https://ffmpeg.org/download.html) installed
+- There must be a `praw.ini` file in the `reddit` folder
+- The `useful_functions` setup must be done (see the setup section below)
+
+# Example `praw.ini` file
+
+```{praw.ini}
+[bot1]
+client_id=TYPE_YOUR_CLIENT_ID_HERE
+client_secret=TYPE_YOUR_CLIENT_SECRET_HERE
+password=TYPE_YOUR_PASSWORD_HERE
+username=TYPE_YOUR_USERNAME_HERE
+user_agent=bot1
+```
+
+Details can be found in the
+[PRAW documentation](https://praw.readthedocs.io/en/stable/getting_started/configuration/prawini.html).
+
+# Setup
+
+## Setting up the Python environment
+
+If using Python venv, you can use the code below to set up the code:
+
+```{bash}
+python3 -m pip install -r requirements.txt
+```
+
+If using Conda, you can use:
+
+```{bash}
+conda create --name <env> --file requirements.txt
+conda activate <env>
+conda install pip
+```
+
+The scripts `reddit_scraper.py`, `audio_extraction.py`, and `vad.py` require
+imports from the folder `useful_functions`. Here are the steps to make the
+imports work:
+
+- Activate the Python environment
+- Change directories into the `data-collection` folder
+- Run `pip install -e .`
+
+The imports should work afterwards.
```

Comment:
this and the next line are worded weird

Reply:
I've [clarified what `pip install -e .`](https://github.com/myHaiven/data-collection/pull/4/commits/a07b72d347d9e9e1e597d38fd12b176064d5f7cc) does and fixed up the wording.